### ISSUE TITLE
L1 loss and EPE become extremely large

### ISSUE THREAD

**Issue by shanyuhu** [NVIDIA/flownet2-pytorch#38]
Hi, @fitsumreda, thanks for your excellent work.
I train Flownet2 under your default settings with 4 Gpus on Sintel dataset, but the L1 loss and EPE become quite larger after several epochs. what is wrong with the training process?
The training command is 
“python main.py --batch_size 8 --model FlowNet2 --loss=L1Loss --optimizer=Adam --optimizer_lr=1e-4 --training_dataset MpiSintelFinal --training_dataset_root data/sintel/training --validation_dataset MpiSintelClean --validation_dataset_root data/sintel/training“
![qq 20180212194641](https://user-images.githubusercontent.com/15082681/36095510-778ce2f6-102d-11e8-94fb-638583813936.png)

**Comment by fitsumreda** [comment]
The Flownet2 paper actually recommends to train the networks one at a time, starting with FlowNetC, then fix this, and train FlowNetS1 and so on.
training flownet2 directly on flying-chairs would lead to the type of EPE explosion you're getting. 

another approach is to turn on BatchNorm for all layers in flownet2 or significantly reduce your learning rate.

**Comment by shanyuhu** [comment]
Thanks for your reply. I will try it.

**Comment by wtyhub** [comment]
Hi,@shanyuhu, I also get the same problem ,L1 loss and EPE become quite larger after several. Have you solved this problem, can you give me some guidance? how can i train the datasets
I use 1 gpu. pytoch1.2.0, cuda10

python main.py --batch_size 8 --model FlowNet2 --loss=L1Loss --optimizer=Adam --optimizer_lr=1e-4 --training_dataset MpiSintelFinal --training_dataset_root datasets/sintel/training/ --skip_validation

my error:
![image](https://user-images.githubusercontent.com/29724042/72218110-57eb8300-358b-11ea-9f71-9938cb465c19.png)
@fitsumreda
